zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -410,21 +411,20 @@ public void unicastNotebookJobInfo(NotebookSocket conn, Message fromMessage) thr
       .put("notebookJobs", response)));
   }
 
-  public void unicastUpdateNotebookJobInfo(NotebookSocket conn, Message fromMessage)
-      throws IOException {
-    double lastUpdateUnixTimeRaw = (double) fromMessage.get("lastUpdateUnixTime");
-    long lastUpdateUnixTime = new Double(lastUpdateUnixTimeRaw).longValue();
-
+  public void broadcastUpdateNotebookJobInfo(long lastUpdateUnixTime) throws IOException {
     List<Map<String, Object>> notebookJobs;
-    AuthenticationInfo subject = new AuthenticationInfo(fromMessage.principal);
-    notebookJobs = notebook().getJobListforNotebook(false, lastUpdateUnixTime, subject);
+    notebookJobs = notebook().getJobListforNotebookFromUnixTime(false, lastUpdateUnixTime, null);
```

Comment:
if notebookJobs is null then `jobs` in result value packet is empty.
but, i agree for your opinion, i should be add null check logic.
I think it would help we understand the safety of the code and good readability.
Thank you.